dynamicio/config.py

```diff
@@ -52,17 +52,17 @@
             }
         }
 """
-__all__ = ["IOConfig", "SafeDynamicLoader"]
+__all__ = ["IOConfig", "SafeDynamicResourceLoader", "SafeDynamicSchemaLoader"]
```

Comment:
this was the simplest way to implement this. I could try to parameterise the original loader but it turns out it is more complex than it looks. This keeps things cleaner at least.